exploring_UV_extinction_curves_pt1/downloading_UV_data_pt1.ipynb

```diff
@@ -0,0 +1,1433 @@
+{
```

Comment:
<blockquote>are created by representing&nbsp;k(λ−V)</blockquote><p>Some wording: I would change "representing" to "plotting"</p><p> </p><p>For Extinction: can clarify, difference in what magnitudes? Is it the difference between the magnitude of the band when observed through dust vs. not? While it's abbreviated as A_lambda, I don't see that expression in the equation below. Is A_lambda equivalent to something else in the equation? Is it A(lambda)?</p><p> </p><p>Likewise, what is E in the equation? Or, if not worth saying, maybe just remove that part!</p><p> </p><p>There is some markdown error with the Note.</p><p> </p><p>At this point, as the reader, I am wondering, am I going to be making an extinction curve out of a spectrum? Or will I be able to directly download an extinction curve from the archive? So some more clarification of what we will do will help. I know this is one of a two-part tutorial and I think it's okay to say that - something like, in Part 1 you will do this, in Part 2 you will do that. I will defer to the STScI folks for specific style guidelines. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>